packages/genesys-spark-components/src/components/stable/gux-table/gux-all-row-select/gux-all-row-select.tsx

```diff
@@ -14,6 +14,9 @@ import { buildI18nForComponent, GetI18nValue } from '../../../../i18n';
 import { randomHTMLId } from '@utils/dom/random-html-id';
 import tableResources from '../i18n/en.json';
 
+/**
+ * @slot label - input label.
+ */
```

Comment:
Looking for feedback on:

if this should be a slot or a property
if the name label is ok or you have other suggestions.